changelog.md

```diff
@@ -119,6 +119,7 @@ with other backends. see #9125. Use `-d:nimLegacyJsRound` for previous behavior.
 
 - nil dereference is not allowed at compile time. `cast[ptr int](nil)[]` is rejected at compile time.
 
+- `os.copyFile` is now 2.5x faster on OSX.
```

Comment:
But would be nice to mention **how?**

os.copyFile is now 2.5x faster on OSX, by using a faster OSX syscall foobar().

Reply:
done